### DISCUSSION TITLE
Decision tree with pycaret -- TypeError: can only concatenate str (not "numpy.int8") to str

### DISCUSSION THREAD

**Discussion by MiladSh75** [pycaret/pycaret#2908] [General]
I created dt model with thid code with pycaret:
[
from pycaret.classification import *
...
#Decision Tree Classifier
dt = create_model('dt')
]

and when I want to plot tree, I wrote this code:
plot_model(dt, plot="tree")

but I got this error:
TypeError: can only concatenate str (not "numpy.int8") to str

**Comment by tvdboom** [comment]
Please, open a issue with a full reproducible example of the bug. We can't debug any errors without a full reproducible example.
